core/state/statedb.go

```diff
@@ -877,3 +889,34 @@ func (s *StateDB) Commit(deleteEmptyObjects bool) (common.Hash, error) {
 	}
 	return root, err
 }
+
+func (s *StateDB) AddAddrToAccessList(addr common.Address) {
+	if s.accessList.AddAddr(addr) {
+		s.journal.append(accessListAddAccountChange{&addr})
+	}
+}
+
+func (s *StateDB) AddSlotToAccessList(addr common.Address, slot common.Hash) {
+	addrCh, slotCh := s.accessList.AddSlot(addr, slot)
```

Comment:
I'm not sure it's advisable to use the `Ch` suffix to mean `Change`, since it's super ingrained in Go's style as `channel`. Not sure what a better name would be though.

Reply:
Changed it to `Mod` (modification /  modified)